Controllers/WishListsController.cs

```diff
@@ -0,0 +1,256 @@
+            if (selectedList == null) {
+                selectedList = wishLists.SingleOrDefault(wp => wp.IsDefault);
+            }
+            return View(_contentManager.BuildDisplay(selectedList));
+        }
+        [Authorize]
+        public ActionResult Create() {
+            var model = _wishListServices.CreateShape(_wca.GetContext().CurrentUser);
+            return View("WishListEditor", model);
+        }
+        [Authorize]
+        [HttpPost, ActionName("Create")]
+        public ActionResult CreatePost(string new_wishlist_title, int productid = 0) {
+            return (CreateWishList(new_wishlist_title, productid));
+        }
+
+        [Authorize]
+        public ActionResult Edit(int id = 0) {
+            var model = _wishListServices.SettingsShape(_wca.GetContext().CurrentUser, id);
+            return View("WishListsSettings", model);
+        }
+
+        [Authorize]
+        [HttpPost, ActionName("Edit")]
+        public ActionResult EditPost(int id = 0) {
+            return UpdateSettings(id);
+        }
+
+        [Authorize]
+        [HttpPost]
+        public ActionResult Delete(int id) {
+            return RedirectToAction("Index");
+        }
+
+
+        [HttpPost]
+        public ActionResult CreateWishList(string new_wishlist_title, int productid = 0) {
+            var user = _wca.GetContext().CurrentUser;
+            if (user == null) {
+                return new HttpUnauthorizedResult();
+            }
+
+            var wishlistId = 0;
+            //create wish list
+            var wishList = _wishListServices.CreateWishList(user, new_wishlist_title);
+            wishlistId = wishList.ContentItem.Id;
+            //add product to wishlist
+            if (productid > 0) {
+                var productPart = _contentManager.Get<ProductPart>(productid);
+                var productattributes = ParseProductAttributes();
+
+                _wishListServices.AddProductToWishList(user, wishList, productPart, productattributes);
+            }
+
+            return RedirectToAction("Index", new { id = wishlistId });
+        }
+
+        [HttpPost]
+        public ActionResult AddToWishList(int wishListid, int productid) {
+            var user = _wca.GetContext().CurrentUser;
+            if (user == null) {
+                return new HttpUnauthorizedResult();
+            }
+            //get selected wishlist
+            var wishList = _wishListServices.GetWishList(user, wishListid);
+
+            if (productid > 0) {
+                var productPart = _contentManager.Get<ProductPart>(productid);
+                var productattributes = ParseProductAttributes();
+
+                _wishListServices.AddProductToWishList(user, wishList, productPart, productattributes);
+            }
+
+            return RedirectToAction("Index", new { id = wishList.ContentItem.Id });
+        }
+
+        [HttpPost]
+        public ActionResult AddToCart(int productId, int quantity = 1) {
```

Comment:
even then, however, if I hit F12, change the values from that form, and then click "Add to cart", the product that is added has the edited values.
e.g.:
 - my wishlist has a ProductX in Size 56 (attribute and value)
 - I hit F12 and change the valu of the hidden to 52
 - I click on the "Add to cart" button
 - The product added to the cart has Size 52